frontend/src/index.scss

```diff
@@ -112,3 +108,18 @@ $standardMargin: 10px;
 .ag-rich-select-list {
   height: auto !important;
 }
+
+.check-icon {
+  margin-bottom: 5px;
+}
+
+.warning-icon:hover {
+  background-color: rgba(0, 0, 0, 0.15);
+}
+
+.warning-icon {
+  margin-bottom: 5px;
+  padding: 2px;
+  cursor: pointer;
+  border-radius: 4px;
+}
```

Comment:
Because the warning icon became a clickable button, that added padding to the icon. Here, I'm making some cosmetic adjustments to make the check and warning icons properly aligned.